breathe/directives.py

```diff
@@ -246,6 +246,8 @@ class _DoxygenClassLikeDirective(BaseDirective):
         "path": unchanged_required,
         "project": unchanged_required,
         "members": unchanged,
+        "membergroups": unchanged,
```

Comment:
If I understand correctly if the `:membergroups:` option is used you have to pass actual member groups (at least 1). In that case this should be `unchanged_required`. Or is there a valid use of using `:membergroups:` without actually specifying any groups? Perhaps I'm missing something.

If specifying the groups is required the documentation examples should use `:membersgroups: ...` instead of `:membersgroups: [...]`, as the `[]` implies optional.

Reply:
You're right, will change.